copy/src/component/Header/header.jsx

```diff
@@ -0,0 +1,22 @@
+import { NavBar } from "../NavBar/navbar.jsx";
+import { AddLinkBar } from "../AddLinkBar/addlinkbar.jsx";
+import "./header-style.css";
+
+export function Header({ folder, user }) {
+  return (
+    <header className="header-section">
+      <NavBar user={user} />
+      <div className="header-info">
+        <img
+          src={folder.owner.profileImageSource}
+          className="header-profile"
+        ></img>
+        <div className="header-user">
+          {"@"}
+          {folder.owner.name}
```

Comment:
```suggestion
          {`@${folder.owner.name}`}
```
[중요도: 중간] 이렇게 처리하는게 더 깔끔하겠군요. literal을 써보세요~